site/pages/api/admin/update-email-preference.api.ts

```diff
@@ -16,8 +16,6 @@ const updateEmailPreference = async (req: NextApiRequest, res: NextApiResponse)
 
         if (!session) {
             throw new Error("No session found");
-        } else if (!session.isOrgAdmin) {
-            throw new Error("Invalid user accessing the page");
```

Comment:
Just wondering if the file path should be changes if we aren't checking if they are not an admin anymore. It is still api/admin...

Reply:
Perhaps we still want to check groups based on what email preference we are updating